### PR TITLE
SNHUT-4455/LearnersContactTab

### PR DESCRIPTION
**Pull request by noraanazia**
### Description

- Learners contact tab. It has a mailto feature (new tab). Optional tag added after video was made.

**To test:** 
http://localhost:4200/learner/O9uqi8CTgy21CAqmPtqz

#### Video

- Paste your link here.

#### Screenshots

| Original | Updated |
| -------- | ------- |
|          |         |

### Other Details

- [JIRA Ticket](https://collectiveshift.atlassian.net/browse/SNHUT-4455

https://user-images.githubusercontent.com/109235207/227436540-f9d4d0cf-6af6-4979-9f61-47402fdff53c.mov

)

#### Checklist

- [ ] **Assumptions of User Story met (including acceptance criteria)**
- [ ] **All new necessary Unit tests were CREATED** to promote code coverage
- [ ] **All required unit tests are PASSING**
- [ ] The proposed functionality was tested manually locally, and it's working fine.
- [ ] PR was properly reviewed by a team member (at least 1 approval required to merge!)
- [ ] Unnecessary console.logs AND/OR comments were removed
- [ ] Feature ok-ed by UX designer (If needed - not for back end components) and Product Owner
- [ ] Ensure configuration and/or build changes documented

### CODE REVIEW

File: microservices/turbine-admin/apps/toboggan-app/src/app/learners/components/learners-personal-tab/learners-personal-tab.component.ts [hunk 1]
```diff
@@ -104,6 +104,7 @@ export class LearnersPersonalTabComponent implements OnChanges {
           ...learnerPayload
         } = learnerProfilePayload;
 
+        console.log('pay', learnerPayload)
```
Comment (Mubashir-sadaindia) [inline microservices/turbine-admin/apps/toboggan-app/src/app/learners/components/learners-personal-tab/learners-personal-tab.component.ts:107]: remove log

**Review by Mubashir-sadaindia — CHANGES_REQUESTED** [review]

**Review by Mubashir-sadaindia — APPROVED** [review]

**Comment by Mubashir-sadaindia** [comment]
LGTM

**Review by mk-sada — COMMENTED** [review]
https://user-images.githubusercontent.com/107949139/227727064-7542ebf2-22e4-4a7c-b40b-436ed78cbc4a.mp4

- The error messages aren't triggering properly when a user (with mild ADHD-esque symptoms)
  - navigates to an incomplete profile page
  - tries to edit contact
  - changes their mind and tries to navigate away from the edit page

- Trying to save a partially completed form subsequently is also not triggering the error messages (0:48 in the video attached)

**Comment by noraanazia** [comment]
> * The error messages aren't triggering properly when a user (with mild ADHD-esque symptoms)
>   
>   * navigates to an incomplete profile page
>   * tries to edit contact
>   * changes their mind and tries to navigate away from the edit page
> * Trying to save a partially completed form subsequently is also not triggering the error messages (0:48 in the video attached)

Thank you for reviewing this @mk-sada . I have fixed all the issues above. I didn't test with a 'new user' after my last commit. I have done that now. Pls test again using the ADHD-ish user LOL . Thanks again.

**Pull request merged**